### DISCUSSION TITLE
I need some clarification on some things

### DISCUSSION THREAD

**Discussion by FieryMewtwo** [Universal-Team/Universal-Updater#104] [Q&A] [answered]
Specifically, with making a Unistore:

- Do storeContent and storeInfo need to be separate files? And if so, what file extension do I use?

**Comment by Epicpkmn11** [comment, marked as answer]
They should be the same file, there's an example of a small but complete UniStore on the wiki: https://github.com/Universal-Team/Universal-Updater/wiki/unistore-example

And extension should be `.unistore`, not sure if UU has a check so `.json` may work, but `.unistore` is the normal one to use.

**Reply by FieryMewtwo** [reply]
Thank you!
